File: CSCI_150_Hardware/Test_Scripts/particleRequestTest.py
#device 50ff6f065067545626430587
#token  895043e69c01b80b464cdd995e913efa4b25c3a3
#curl "https://api.particle.io/v1/devices/50ff6f065067545626430587/temp?access_token=895043e69c01b80b464cdd995e913efa4b25c3a3"

import requests
import json
r = requests.get("https://api.particle.io/v1/devices/50ff6f065067545626430587/temp?access_token=895043e69c01b80b464cdd995e913efa4b25c3a3")
r.json()
print(r.json())

File: CSCI_150_Hardware/Test_Scripts/particleRequestTest.js
const Http = new XMLHttpRequest();
const url='https://api.particle.io/v1/devices/50ff6f065067545626430587/temp?access_token=895043e69c01b80b464cdd995e913efa4b25c3a3';
Http.open("GET", url);
Http.send();

Http.onreadystatechange = (e) => {
  console.log(Http.responseText)
}
